docs/providers/openstack.md

```diff
@@ -0,0 +1,34 @@
+# OpenStack UniK Provider
```

Comment:
Since this is just a preliminary support for OpenStack, please add a disclaimer saying that this is a work in progress.